app/controllers/application_controller.rb

```diff
@@ -521,6 +523,21 @@ def user_is_banned?
     BannedUserResolver.new(current_user).banned_for_sp?(issuer: current_sp&.issuer)
   end
 
+  def user_account_creation_device_profile_failed?
+    return unless IdentityConfig.store.account_creation_device_profiling == :enabled
```

Comment:
Given that this is a predicate method, wouldn't `return false unless ...` be cleaner?